whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -340,9 +337,16 @@ private RdapObject mapCommonNoticesAndPort(final RdapObject rdapResponse, final
     private RdapObject mapCommonConformances(final RdapObject rdapResponse) {
         rdapResponse.getRdapConformance().addAll(List.of(RdapConformance.CIDR_0.getValue(),
             RdapConformance.LEVEL_0.getValue(), RdapConformance.NRO_PROFILE_0.getValue()));
+        mapRedactionConformance(rdapResponse);
         return rdapResponse;
```

Comment:
I think we should directly add redaction in conformance , it is the tag that tells the user that we are following the redaction RFC. So if fields are not included or changed you will see redacted element

Reply:
I this that the RFC says to add just the redaction when the removal is applied I will double check

Reply:
I will add the conformance I don't see anything in the rfc now

Reply:
Done

Reply:
Sorry to unresolve this, I read RFC and I did not see anything specific . I would like @eshryane  suggestions on this .

Reply:
yes, I saw nothing in the RFC related to this as well

Reply:
@maggarwal13 I'm OK to always add the redaction profile in conformance